src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -79,12 +79,10 @@ abstract contract DelegationManagerStorage is IDelegationManager {
     mapping(address => mapping(bytes32 => bool)) public delegationApproverSaltIsSpent;
 
     /**
-     * @notice Minimum delay enforced by this contract for completing queued withdrawals. Measured in blocks, and adjustable by this contract's owner,
-     * up to a maximum of `MAX_WITHDRAWAL_DELAY_BLOCKS`. Minimum value is 0 (i.e. no delay enforced).
-     * @dev Note that the withdrawal delay is not enforced on withdrawals of 'beaconChainETH', as the EigenPods have their own separate delay mechanic
-     * and we want to avoid stacking multiple enforced delays onto a single withdrawal.
+     * @notice Deprecated from an old Goerli release, setting withdrawalDelayBLocks on per strategy basis
```

Comment:
Clarify comment:

"Deprecated from an old Goerli release, which had a global withdrawal delay for all withdrawals. We now configure withdrawal delays on a per strategy basis"

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/392/commits/323fa4e387941179468d541a4c9d9470c2382bd1#diff-4d12f7b9016c76f80ea4d90dc4e49dbddda5177ec6d7c8cd4055b2f3f857a20aR84